release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -809,3 +816,9 @@ the client to fail or incorrectly patch the RHV machine status.
 * {product-title} cluster autoscaling on RHV fails due to cluster resource
 computation errors.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822118[*BZ#1822118*])
+
+* When a user selects a node or a group of nodes in the *Topology* view on the Firefox browser, the backgrounds of all the associated labels and nodes become transparent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822337[*BZ#1822337*])
```

Comment:
s/When a user/When using the Firefox browser to select a node/
s/view on the Firefox browser,/view,/

Reply:
Fixed